lib/applications/resolvers.ts

```diff
@@ -40,6 +40,7 @@ import {
   UpdateApplicationReasonForReplacementResult,
 } from '@lib/graphql/types';
 import { flattenApplication } from '@lib/applications/utils';
+import { MutationUpdateNewApplicationGeneralInformationArgs } from '@lib/graphql/types';
```

Comment:
Add this import to the list of imports from `@lib/graphql/types` above to keep things clean